README.md

```diff
@@ -265,8 +265,8 @@ Code blocks can also be added:
       | B { 'B' }
       ;
 
-When the terminal `A` would be processed the returned value would be "B", for
-terminal `B` the returned value would be "B".
+When the terminal `A` would be processed the returned value would be `'A'`, for
+terminal `B` the returned value would be `'B'`.
```

Comment:
Sure?, I wanted to emphasize that I was taking about the strings at the block not the non-terminals.